android/java/org/chromium/chrome/browser/shields/BraveShieldsHandler.java

```diff
@@ -811,8 +812,10 @@ public void onClick(View view) {
 
     private void setUpReportBrokenSiteLayout() {
         TextView mReportSiteUrlText = mReportBrokenSiteLayout.findViewById(R.id.report_site_url);
-        final Uri reportUri = Uri.parse(mUrlSpec).buildUpon().clearQuery().build();
-        mReportSiteUrlText.setText(reportUri.toString());
+        Uri reportUri = Uri.parse(mUrlSpec).buildUpon().clearQuery().build();
```

Comment:
nit: why `final` modifier is dropped here? It looks it is assigned just once.

Reply:
we needed final earlier as we were using `reportUri` inside a callback and for that he has to be final. but with the current change, it's not required.